modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,64 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The following table outlines supported features of primary and secondary networks created by `UserDefinedNetwork` (UDN) and other secondary networks. For more information on, the limitations of user-defined networks see: xref:../networking/ . For more information on secondary networks that are created through network attachement defintions using multus, see xref:../networking/.
+
+// column 4 and 8 will need put back in once supported
+[cols="<.^3,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1"]
+|====
+|Network type
+|definition
+|role
+//| services
+|east-west traffic
+|north-south traffic
+|Persistent IPs
+//|`NetworkPolicy`
+|`MultiNetworkPolicy`
+|Ingress
+|Egress IP
+
+|UDN primary
+|A network that is created by the user using the UDN custom resource definition (CRD) and declared as primary.
+|Administrator and developer
+//|X [^1^] [^2^]
+|X [^1^] [^2^]
+|X [^1^] [^2^]
+|X [^1^]
+//|X [^1^] [^2^]
+|
+|X [^1^] [^2^]
+|
+
+|UDN Secondary
+|A network that is created by the user using the UDN (CRD) and declared as secondary.
+|Administrator and developer
+//|X
+|X [^1^] [^2^]
```

Comment:
this is gonna get harder with the admin CRDs cause there we do support localnet or plan to then overlay will have support, would be good to note this down for 4.18 docs this needs to change

Reply:
The footnotes have `localnet` and I thought we could do CUDN as another row if necessary.

Reply:
Or perhaps we add CUDN as a footnote.